charts/telemeter-reporting/templates/report-queries/nodes-by-type.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "nodes-by-type-raw"
+  labels:
+    telemeter-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportDataSources:
+  - "nodes-by-type"
+  columns:
+  - name: node_count
+    type: double
+    unit: nodes
+  - name: node_seconds
```

Comment:
Do we actually need this column?

Reply:
I don't like that node_count doesn't take into account how long the report period is -- but perhaps this might be better to move into exclusively the non-raw version of the query?